### PR TITLE
Button

### PR DESCRIPTION
**Pull request by deadlybutter**
#### What's this PR do?
- Adds a Facebook login button
- Hides it behind a query string (`/login?fb=true`)
- Removes unused code

#### How should this be reviewed?
Click the button, magic should happen!
Also, does it look okay across mobile --> desktop?

⚠️ ⚠️ ⚠️ 
This changes the `package.json` to allow me to develop locally while we figure out what is going on. do not merge until this + the package lock changes are committed back out.

### CODE REVIEW

**Comment by deadlybutter** [comment]
And some screenshots!

screen shot 2017-07-19 at 3 35 51 pm
screen shot 2017-07-19 at 3 40 56 pm

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -54,6 +54,10 @@
     </div>
 
     <div class="container__block -centered">
+        @if (app('request')->input('fb') === 'true')
```
Comment (DFurnes) [inline resources/views/auth/login.blade.php]: You can replace `app('request')` with `request()`, and should this be checking `query` explicitly?

```blade
@if (request()->query('fb') === 'true')
   <!-- ... -->
@endif
```

Comment (deadlybutter) [inline resources/views/auth/login.blade.php]: 🌮

**Comment by deadlybutter** [comment]
retested w/ the template changed & cleaned up the package junk

**Pull request merged**